pkg/metrics/constants.go

```diff
@@ -28,10 +28,11 @@ const (
 	ProvisionerLabel = "provisioner"
 
 	// Reasons for CREATE/DELETE shared metrics
-	ConsolidationReason = "consolidation"
-	ProvisioningReason  = "provisioning"
-	ExpirationReason    = "expiration"
-	EmptinessReason     = "emptiness"
+	DeprovisioningReason = "deprovisioning"
+	ConsolidationReason  = "consolidation"
+	ProvisioningReason   = "provisioning"
+	ExpirationReason     = "expiration"
+	EmptinessReason      = "emptiness"
```

Comment:
Thoughts on collapsing emptiness/consolidation into the same concept from a customer perspective. Essentially, emptiness is a consolidation action where 0 pods are migrated to existing capacity. Perhaps we can call this "underutilized"?

Reply:
This is on my backlog, but I'll prioritize it after refactoring, expiration, and Drift.

Reply:
sgtm